### ISSUE TITLE
[HELP NEEDED] Folder full of "selective sync conflict"

### ISSUE THREAD

**Issue by raffaem** [SamSchott/maestral#348]
I followed the procedure in #346 

> Quit Maestral.
> 
> Moved the Dropbox folder out in new partition.
> 
> Formatted the old partition.
> 
> Moved Dropbox folder back in.
> 
> The old partition is mounted at a different mountpoint (e.g. /media/user/intdata instead of /media/user/data).
> 
> Opening Maestral, it complains that the folder cannot be found anymore (of course it keeps looking for /media/user/data).
> 
> It asks to "unlink" the account or insert a location to (re-download?) the folder.
> 
> I put in the new location of the Dropbox folder (in /media/user/intdata).
> 
> Click Ok.
> 
> Maestral shows that the folder is not empty (I don't remember the exact dialogs now....), and asks what you want to do with it. I pressed "Merge". Then Maestral asks which remote folders you want to sync, and all remote folders are selected, even though only a few of the remote folders are in the local folder. I manually unselected the remote folders that are not in local.

Now my Dropbox _local_ folder has "(selective sync conflict)" appended to the folders _I chose not to sync_ because were _not in the original Dropbox local folder_.

What do I have to do with those folders? Can I remove them?

**Comment by SamSchott** [comment]
Oops. It appears that your selective sync settings where indeed remembered / applied correctly but for some reason they where ignored during the sync when merging the folder. As I said in #346, the logs would be very helpful here.

Once that initial sync was completed, Maestral detected that local folders are present which have the same name as excluded folders on the remote Dropbox and created selective sync conflict for each of them. This is the correct and expected behaviour.

Those folders should be safe to delete, either locally or on Dropbox by just searching for "selective sync conflict", they are merely duplicates of the original folders. Just verify that the original folders are present on Dropbox and contain the data you expect before deleting anything...

**Comment by raffaem** [comment]
If I remove the "selective sync conflict" folders from my local storage, Maestral re-adds them.

Maybe I should wait until it finishes to sync?

**Comment by SamSchott** [comment]
> Maybe I should wait until it finishes to sync?

Yes. As long as parts of those folders are still syncing, they will be recreated.

**Comment by raffaem** [comment]
I have change my keyring daemon from gnome-keyring to KeePassXC

Maestral asked for a new API key from Dropbox website.

Before confirming the folder I have deleted the "selective sync" folders from both Dropbox remote folder (through the web interface) and the local folder.

Then I went to Maestral again and confirmed the old folder location.

Maestral is now syncing and the logs are like
```
Modification of "<REDACTED>" detected but file content is the same as on Dropbox
```

Let's hope all will go well :) I have full backups of course

------

I think I am putting your software through a torture test. I hope they will not jail me for that :)

I will buy you a beer if I ever go to UK!
